docker-compose.yml

```
@@ -35,71 +44,32 @@ services:
       - "9601:9601"
       - "8545:8545"
 
-  cockroachdb:
-    # Only use cockroachdb single-node clusters for non-production environment
-    image: cockroachdb/cockroach:latest-v23.1
-    command: start-single-node --insecure --store=attrs=ssd,path=/var/lib/cockroach/,size=20%
-    restart: "no"
-    environment:
-      - COCKROACH_DATABASE=nakama
-      - COCKROACH_USER=root
-      - COCKROACH_PASSWORD=${DB_PASSWORD:-development}
-    volumes:
-      - data:/var/lib/cockroach
-    expose:
-      - "8080"
-      - "26257"
-    ports:
-      - "26257:26257"
-      - "8080:8080"
-    healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8080/health?ready=1"]
-      interval: 3s
-      timeout: 3s
-      retries: 5
-
-  redis: # This doesn't have the correct persistence settings. Don't use on for prod.
-    image: redis:latest
-    command: redis-server # TODO: This runs without password. Don't use for prod.
-    expose:
-      - "6379"
-    ports:
-      - "6379:6379"
-    restart: always
-
-  cardinal:
-    build: ./cardinal
-    depends_on:
-      - redis
-    expose:
-      - "3333"
-    ports:
-      - "3333:3333"
-    environment:
-      - CARDINAL_PORT=3333
-      - CARDINAL_NAMESPACE=${CARDINAL_NAMESPACE}
-      - REDIS_ADDR=redis:6379
-      - REDIS_MODE=normal
-    restart: unless-stopped
-
   nakama:
     platform: linux/amd64
     image: us-docker.pkg.dev/argus-labs/world-engine/relay/nakama@sha256:60737f1de75b5e1dfe0f1eb557ebf6f8c691cc2812950dc4e3132242709ddc09
+    container_name: nakama
     depends_on:
-      cockroachdb:
+      nakama-db:
         condition: service_healthy
       cardinal:
         condition: service_started
     environment:
       - CARDINAL_ADDR=${CARDINAL_ADDR:-cardinal:3333}
-      - CARDINAL_NAMESPACE=world
-      - DB_PASSWORD=${DB_PASSWORD:-development}
+      - CARDINAL_NAMESPACE=${CARDINAL_NAMESPACE}
+      - DB_PASSWORD=${DB_PASSWORD:-very_unsafe_password_replace_me}
     entrypoint:
       - "/bin/sh"
       - "-ecx"
       - >
-        /nakama/nakama migrate up --database.address root:$DB_PASSWORD@cockroachdb:26257/nakama &&
-        exec /nakama/nakama --config /nakama/data/local.yml --database.address root:$DB_PASSWORD@cockroachdb:26257/nakama
+        /nakama/nakama migrate up --database.address postgres:$DB_PASSWORD@nakama-db:5432/nakama &&
+        exec /nakama/nakama --config /nakama/data/config.yml --database.address postgres:$DB_PASSWORD@nakama-db:5432/nakama
```

escaped command & changed the default local config file name.
```suggestion
        exec /nakama/nakama --config /nakama/data/local.yml --database.address postgres:$$DB_PASSWORD@nakama-db:5432/nakama
```